_data/speakers.yml

```diff
@@ -63,18 +63,18 @@
   subtitle: Speaker
   twitter: MoAkhshik
   github: makhshik
-  web:
+  web: 
   image: mo-akhshik.jpg
   thumbnail: mo-akhshik.jpg
   alt: Mo
-  topic:
-  proposal_summary:
+  topic: A Novel Bayesian Model to Fit Spectrophotometric Data of Hubble and Spitzer Space Telescopes
+  proposal_summary: Understanding how the most massive galaxies rapidly formed and quenched when Universe was only $\sim$3 billion years old is one of the major challenges of extragalactic astronomy. In this talk, I will discuss how to improve our understanding of massive galaxy formation by combining the spectro-photometric observations of the Hubble and Spitzer Space Telescopes for strong gravitationally lensed galaxies. In particular, a multi-level regression model is built that can fit all multi-wavelength data for a range of instruments within a hierarchical Bayesian framework to constrain the properties of the stellar populations. The details of how this model is implemented using \texttt{pymc3}, as well as the estimates of the posteriors of all parameters of interest and nuisance parameters will be highlighted.
```

Comment:
can you remove the latex things? Unluckily they won't work on the website and will look weird:

* `$\sim$3` -> `~`
* `\texttt{pymc3}` -> PyMC3

Reply:
Changed.